### DISCUSSION TITLE
Which CPU Temp source do i needs to use for my 5950x

### DISCUSSION THREAD

**Discussion by DrMental** [Rem0o/FanControl.Releases#1952] [Q&A] [answered]
![Screenshot 2023-07-29 201023](https://github.com/Rem0o/FanControl.Releases/assets/140925436/4d60d142-ea88-4c78-8ce9-59e21cf7257b)

I'm trying to make a fan curve

**Comment by Rem0o** [comment, marked as answer]
CCDs Average, CCDs Max or Core will work. Just adjust your curve around that specific sensor's range.

**Reply by DrMental** [reply]
I was a bit confused because I was looking for CPU Package 

I'm watching JayzTwoCents vid

**Reply by Rem0o** [reply]
Jay's video uses an Intel CPU. AMD cpus have different sensors.

**Reply by kklittich** [reply]
For my Ryzen 3600 I used the average of CCD  and core. I initially pulled in sensors from HWInfo plugin which also shows a ‘hot spot’ temp, but it seemed overkill to use that (and I didn’t really need HWInfo so removed the plugin). It is working great, very happy with the project.
